docs/gno-infra/gno-infra.md

```diff
@@ -0,0 +1,9 @@
+---
+id: gno-infra
+---
+
+# Gno Infra
```

Comment:
I dislike "Infra".

Can we use something else, like "Infrastructure", "Hosting", or "Node Operator"?

Reply:
Changed it to "Gno Infrastructure": [f33296d](https://github.com/gnolang/gno/pull/2212/commits/f33296d483063602460c8fa424e38142aaef4569)